### ISSUE TITLE
Unable to create an update_policy

### ISSUE THREAD

**Issue by timstoop** [terraform-google-modules/terraform-google-vm#227]
### TL;DR

When trying to create a stateful mig, I need to set the update_policy. However, it seems to require all settings to be added, even conflicted ones. When adding the conflicting setting, it of course errors on rollout.

### Expected behavior

I would expect it to require either `max_surge_fixed` or `max_surge_percent`, and either `max_unavailable_fixed` or `max_unavailable_percent`.

### Observed behavior

When I add only `max_surge_fixed` and `max_unavailable_fixed`, I get the error during `plan`:

> The given value is not suitable for child module variable "update_policy" defined at .terraform/modules/mig_stateful/modules/mig/variables.tf:77,1-25: element 0: attributes "max_surge_percent" and "max_unavailable_percent" are required.

Adding those, gives I get an error during `plan`:

> Error: Conflicting configuration arguments
>
> with module.mig_stateful.google_compute_region_instance_group_manager.mig,
>   on .terraform/modules/mig_stateful/modules/mig/main.tf line 38, in resource "google_compute_region_instance_group_manager" "mig":
>   38: resource "google_compute_region_instance_group_manager" "mig" {
>
> "update_policy.0.max_surge_fixed": conflicts with update_policy.0.max_surge_percent

And the same for the three other settings.

### Terraform Configuration

```hcl
module "mig_stateful" {
  source  = "terraform-google-modules/vm/google//modules/mig"
  version = "7.4.0"

  project_id                = module.project-factory.project_id
  region                    = var.region
  target_size               = 1
  hostname                  = "stateful-vm"
  instance_template         = module.instance_template_stateful.self_link
  distribution_policy_zones = ["europe-west4-a"]

  update_policy = [{
    instance_redistribution_type = "NONE"
    replacement_method = "RECREATE"
    max_surge_fixed = 0
    max_unavailable_fixed = 1
    max_surge_percent = 0
    max_unavailable_percent = 100
    min_ready_sec = 300
    minimal_action = "REPLACE"
    type = "OPPORTUNISTIC"
  }]

  stateful_disks = [{
    device_name = "stateful-root"
    delete_rule = "NEVER"
  }]
}
```

### Terraform Version

```sh
Terraform v1.1.3
on linux_amd64
+ provider registry.terraform.io/hashicorp/google v3.90.1
+ provider registry.terraform.io/hashicorp/google-beta v4.8.0
+ provider registry.terraform.io/hashicorp/null v3.1.0
+ provider registry.terraform.io/hashicorp/random v3.1.0
+ provider registry.terraform.io/hashicorp/time v0.7.2
```

**Comment by quulah** [comment]
Encountered the same thing. Looks like you can workaround it by setting the conficting values to `null`.

Same as this: https://github.com/terraform-google-modules/terraform-google-vm/issues/156
